core/src/jsMain/kotlin/WebBluetoothAdvertisement.kt

```diff
@@ -0,0 +1,10 @@
+package com.juul.kable
+
+import com.benasher44.uuid.Uuid
+import org.khronos.webgl.DataView
+
+public interface WebBluetoothAdvertisement : Advertisement {
+    public val identifier: String
+    public fun serviceDataAsDataView(uuid: Uuid): DataView?
+    public fun manufacturerDataAsDataView(companyIdentifierCode: Int): DataView?
```

Comment:
Ditto, would be nicer without the redundant `AsDataView` type suffix naming?

Reply:
The superinterface `Advertisement` already has a `manufacturerData` function.